test/unit/v1.0.0/converter-v1-to-v21.test.js

```diff
@@ -494,14 +486,324 @@ describe('v1.0.0 to v2.1.0', function () {
                             schema: 'https://schema.getpostman.com/json/collection/v2.1.0/collection.json'
                         },
                         item: [{
-                            auth: null,
                             item: []
                         }]
                     });
                     done();
                 });
             });
         });
+
+        describe('with missing properties', function () {
```

Comment:
Need test for missing `legacy` properties with `.auth`

Reply:
Exists